src/plugins/intel_npu/src/backend/src/zero_device.cpp

```diff
@@ -20,6 +20,25 @@ ZeroDevice::ZeroDevice(const std::shared_ptr<ZeroInitStructsHolder>& initStructs
     zeroUtils::throwOnFail("zeDeviceGetProperties",
                            zeDeviceGetProperties(_initStructs->getDevice(), &device_properties));
 
+    // Query PCI information
+    // Older drivers do not have this implementend. Linux driver returns NOT_IMPLEMENTED, while windows driver returns
+    // zero values. If this is detected, we populate only device with ID from device_properties for backwards
+    // compatibility
+    try {
+        pci_properties.stype = ZE_STRUCTURE_TYPE_PCI_EXT_PROPERTIES;
+        zeroUtils::throwOnFail("zeDevicePciGetPropertiesExt",
+                               zeDevicePciGetPropertiesExt(_initStructs->getDevice(), &pci_properties));
```

Comment:
Can you consider checking a status and if `failed` to fallback to the `else` branch. It would look more natural than catching an exception in my opinion.